package.json

```diff
@@ -18,7 +18,7 @@
     "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry",
     "lint": "yarn eslint && yarn stylelint",
     "eslint": "eslint .",
-    "stylelint": "lerna run stylelint",
```

Comment:
[see the way this was configured previously](https://github.com/okta/odyssey/pull/929/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L17) for more detail around the reason invalid styles got past our lint rules in CI/CD. My change to use `lerna run` was an intermediate step until I could get this PR in!